lib/features/modeling/behavior/ConnectEventBasedGatewayBehavior.js

```diff
@@ -0,0 +1,51 @@
+import inherits from 'inherits';
+
+import CommandInterceptor from 'diagram-js/lib/command/CommandInterceptor';
+
+import { is } from '../../../util/ModelUtil';
+
+import {
+  filter
+} from 'min-dash';
+
+
+/**
+ * Behavior for deleting boundaries from receive task after connecting them with event based gateway
+ */
+export default function ConnectEventBasedGatewayBehavior(
+    eventBus, modeling, bpmnRules) {
```

Comment:
`bpmnRules` is actually never used, so I think we should remove it before merge.